stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/reactions/edit/internal/EditReactionsView.kt

```diff
@@ -110,7 +110,7 @@ public class EditReactionsView : RecyclerView {
         this.reactionsViewStyle = editReactionsViewStyle
         this.bubbleDrawer = EditReactionsBubbleDrawer(reactionsViewStyle)
 
-        reactionsColumns = editReactionsViewStyle.reactionsColumn
+        reactionsColumns = minOf(ChatUI.supportedReactions.reactions.size, editReactionsViewStyle.reactionsColumn)
```

Comment:
Without this fix the bubble's width is enough 5 reactions, but there are only 4 supported reactions defined in `ChatUI.supportedReactions`.